client/webpack-hot-middleware-client.js

```diff
@@ -41,7 +41,8 @@ webpackHotMiddlewareClient.subscribe((obj) => {
   const fn = handlers[obj.action]
   if (fn) {
     const data = obj.data || []
-    fn(...data)
+    const route = data[0].replace(/[index]*\.json$/, '')
```

Comment:
@nkzawa 
I had to do this because now these files routes have `.json` extension.
Do you know this has changed?
Is there any better way to deal with this?